libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -663,6 +663,15 @@ AP_RangeFinder_Backend *RangeFinder::find_instance(enum Rotation orientation) co
     return nullptr;
 }
 
+float RangeFinder::distance_m_orient(enum Rotation orientation) const
```

Comment:
Why are we leaving the `distance_cm_orient` (and `distance_cm`)?   Why not just be consistent and always get metres?

Reply:
It turns out there are many uses of distance_cm. Replacing all of them is a non-trivial amount of effort.

Reply:
maybe I am overestimating it. Please discuss at the dev call and let me know (I am not sure if I can make it tomorrow)